components/Funders.jsx

```diff
@@ -8,7 +8,7 @@ function Partners({ funders }) {
   const theme = useContext(themeContext);
 
   const handleSearch = (e) => {
-    const term = e.target.value.trim().toLowerCase();
```

Comment:
Same issue here too, @omarY23 can you please fix this to both places?

Reply:
> Same issue here too, @omarY23 can you please fix this to both places?

Yes @tanya17-05 . I think I got why this issue is popping up. I am on it. Thanks for guiding.

Reply:
I have made the requested changes @tanya17-05 . I am sure this will fix.